hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -1184,6 +1185,46 @@ final Stream<DynamicTest> claimWithSenderAccount() {
                         .hasKnownStatus(INVALID_SIGNATURE)));
     }
 
+    @HapiTest
+    @DisplayName("a hollow account with maxAutoAssociation 1 as receiver should be successful")
+    final Stream<DynamicTest> hollowNoMaxAutoAssociationSuccess() {
+        final String ALICE = "ALICE";
+        final String BOB = "BOB";
+        return hapiTest(
+                cryptoCreate(ALICE).balance(ONE_HUNDRED_HBARS),
+                createFT(FUNGIBLE_TOKEN_1, ALICE, 1000L),
+                createHollow(1, i -> BOB),
+                getAliasedAccountInfo(BOB)
+                        .has(accountWith().maxAutoAssociations(-1).hasEmptyKey().noAlias()),
+                cryptoUpdate(BOB).sigMapPrefixes(uniqueWithFullPrefixesFor(BOB)).maxAutomaticAssociations(1),
+                tokenAirdrop(moving(1, FUNGIBLE_TOKEN_1).between(ALICE, BOB)).payingWith(ALICE),
```

Comment:
This record should have an auto association

Reply:
This is CLAIM_34 in this doc, https://limechain.notion.site/Token-Claim-Airdrop-Test-Plan-d1cc52e193b147dabe3232484e7f4d59, set Bob maxAutoAssociations to 1, no no tokenClaimAirdrop needed need after airdrop.